src/github/github-context.ts

```diff
@@ -32,10 +33,5 @@ export function getPullRequestNumber(): number | undefined {
 }
 
 export function relativizePath(path: string): string {
-  let length = process.env.GITHUB_WORKSPACE?.length
-  if (!length) {
-    length = 'undefined'.length
-  }
-
-  return path.substring(length + 1)
```

Comment:
use __dirname instead,
this accomodates for running inside docker container.

untested when released as marketplace action.
this has been verified working using checkout